download/src/lib.rs

```diff
@@ -269,8 +269,6 @@ pub mod reqwest_be {
     use std::time::Duration;
 
     use anyhow::{anyhow, Context, Result};
-    #[cfg(feature = "reqwest-rustls-tls")]
```

Comment:
Please just pub a `any` in here with the additional feature that requires rustls.